.env.template

```diff
@@ -4,9 +4,9 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=
-TRELLO_API_SECRET=
-TRELLO_BOARD_NAME=
-
-#used for e2e test, so you leave it blank
-TRELLO_BOARD_ID=
+MONGODB_COLLECTIONNAME=
+MONGO_CONNECTION_STRING=
+dockerPassword= 
+dockerUsername=
+HEROKU_API_KEY=
+HEROKU_USERNAME=
```

Comment:
Although it's good to flag these somewhere, this file might not be quite the right place since the variables aren't relevant for the application itself! The Readme might be a better shout for mentioning these

Reply:
Thank You, Given that application has been moved to Azure Cloud I will remove this and tidy in Mod 12.